cmd/gotk/uninstall.go

```diff
@@ -102,25 +103,31 @@ func uninstallCmdRun(cmd *cobra.Command, args []string) error {
 			kustomizev1.KustomizationKind,
 			sourcev1.GitRepositoryKind,
 			sourcev1.HelmRepositoryKind,
+			helmv2.HelmReleaseKind,
 		} {
-			command := fmt.Sprintf("kubectl -n %s delete %s --all --timeout=%s %s",
+			command := fmt.Sprintf("kubectl -n %s delete %s --all --ignore-not-found --timeout=%s %s",
 				namespace, kind, timeout.String(), dryRun)
 			if _, err := utils.execCommand(ctx, ModeOS, command); err != nil {
-				return fmt.Errorf("uninstall failed")
+				return fmt.Errorf("uninstall failed %s", err.Error())
 			}
 		}
 	}
 
-	kinds := "namespace,clusterroles,clusterrolebindings"
+	var kinds []string
 	if uninstallCRDs {
-		kinds += ",crds"
+		kinds = append(kinds, "crds")
 	}
 
+	kinds = append(kinds, "clusterroles,clusterrolebindings,namespace")
```

Comment:
Wondering if it would be better to split this up into groups, e.g. `["crds", "cluserroles,clusterrolebindings", "namespace"]`.